### ISSUE TITLE
Running Evaluation 

### ISSUE THREAD

**Issue by dhkhey** [huggingface/setfit#83]
Hi,
Thanks for sharing this work.
I am wondering if it is possible to run evaluation dataset to tune hyperparameters. 
The SetFitTrainer doesn't seem to accept arguments like 'evaluation_strategy', 'save_strategy', 'compute_metrics', etc. 
Or perhaps Im doing something wrong?
Thanks.

**Comment by Mouhanedg56** [comment]
Hi @lewtun 
I'll be more than happy to add hyperparameter search in a PR.

**Comment by Mouhanedg56** [comment]
Yeah, I noticed the huge difference between `setfit.Trainer` and `transformers.Trainer`.
As you mentioned, we need to introduce `model_init` parameter in args before adding hyperparameter search.
Can I PR this?

**Comment by lewtun** [comment]
@Mouhanedg56 yes, feel free to start with `model_init` :)

**Comment by Mouhanedg56** [comment]
Thanks @lewtun for approving #87 . Now with `model_init` supported, we can go forward and add a new PR for hyperparameter search.
